song_translator/track/views.py

```diff
@@ -19,14 +22,40 @@ class TrackDetail(generics.RetrieveUpdateDestroyAPIView):
     permission_classes = [permissions.IsAuthenticatedOrReadOnly, IsOwnerOrReadOnly]
 
 
-class TrackTranslate(generics.RetrieveUpdateDestroyAPIView):
+@api_view(['GET', 'PUT', 'DELETE'])
```

Comment:
how do you create translations?

Reply:
when we do not have the  translation, it is automatically created. That is, if nothing is found in the get request, an automatic translation is created